.github/neovim/init.lua

```diff
@@ -0,0 +1,37 @@
+-- Bootstrap lazy.nvim
+local lazypath = vim.fn.stdpath("data") .. "/lazy/lazy.nvim"
```

Comment:
You don't need `lazy`; just clone and set up nvim-treesitter directly.

You might also want to consider using the `main` branch for tests; that is simpler to set up and has much better performance (but still WIP), see https://github.com/tree-sitter/tree-sitter/blob/master/.github/workflows/nvim_ts.yml

Make sure you include a `generate` step!

(You can call `./scripts/install-parsers.lua nu` to only install your parser.)

Check out https://github.com/tree-sitter/workflows as well.

Reply:
I am not happy with the pipeline, you reinforced this gut feeling. I will take a look, thanks!